machine/wnc/wnc_model/installer.conf

```diff
@@ -0,0 +1,9 @@
+# WNC ONIE installer configuration
+
+description="WNC-X86_64"
+
+# Default ONIE block device
+install_device_platform()
+{
+    echo /dev/sda
```

Comment:
This is always be true, if USB disks is plugged, it will always be /dev/sdb, dev/sdc, and so on.